app/src/main/java/org/oppia/app/player/state/CustomCheckbox.kt

```diff
@@ -0,0 +1,69 @@
+package org.oppia.app.player.state
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.text.Spanned
+import android.view.View
+import android.widget.CheckBox
+import android.widget.CompoundButton
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.content.ContextCompat
+import org.oppia.app.R
+import org.oppia.util.data.UrlImageParser
+import android.content.res.ColorStateList
+import android.os.Build
+import android.view.Gravity
+
+// TODO(#163): Move this to a StateCardFragment Low-fi PR.
+/** Custom Checkbox for MultipleSelectionInputInteractionView. */
+class CustomCheckbox(context: Context, private val optionContents: String) : CheckBox(context) {
+
+  // Update default attributes of ItemSelectionInputInteractionView here.
+  init {
+    val paddingPixel = 2
+    val density = resources.displayMetrics.density
+    val paddingDp = (paddingPixel * density).toInt()
+    gravity = Gravity.LEFT
+    setTextColor(ContextCompat.getColor(context, R.color.oppiaDarkBlue))
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      setButtonTintList(
+        ColorStateList(
+          arrayOf(
+            intArrayOf(android.R.attr.state_enabled)
+          ),
+          intArrayOf(ContextCompat.getColor(context, R.color.oppiaDarkBlue))
+        )
+      );
+    }
+    setHighlightColor(ContextCompat.getColor(context, R.color.oppiaDarkBlue))
+    textSize = 16f
+    setPadding(paddingDp, paddingDp, paddingDp, paddingDp)
+    id = View.generateViewId()
+    text = convertHtmlToString(optionContents, rootView).toString()
+    setOnCheckedChangeListener(CompoundButton.OnCheckedChangeListener { buttonView, isChecked ->
+      val msg = "You have " + (if (isChecked) "checked" else "unchecked") + " this Check it Checkbox."
+      Toast.makeText(context, msg, Toast.LENGTH_SHORT).show()
+    })
+  }
+
+  fun convertHtmlToString(rawResponse: Any?, rdbtn: View): Spanned {
```

Comment:
Could we move this to a general-purpose utility with dedicated tests? It seems shared with the custom radio button.

Reply:
moved html parser code  to utility